internal/controller/object/object.go

```diff
@@ -87,17 +88,19 @@ const (
 	errGetConnectionDetails = "cannot get connection details"
 	errGetValueAtFieldPath  = "cannot get value at fieldPath"
 	errDecodeSecretData     = "cannot decode secret data"
+	errSanitizeSecretData   = "failed sanitizing secret data"
```

Comment:
```suggestion
	errSanitizeSecretData   = "cannot sanitizing secret data"
```

nit: for consistency with other errors.